### ISSUE TITLE
New release to fix Node.js 14 warnings

### ISSUE THREAD

**Issue by gengjiawen** [shelljs/shelljs#991]
Fixed in https://github.com/shelljs/shelljs/commit/05374a749992969bb6b853095f4c9308df69aa82.

Current warning
```console
(node:32597) Warning: Accessing non-existent property 'cat' of module exports inside circular dependency
    at emitCircularRequireWarning (internal/modules/cjs/loader.js:814:11)
    at Object.get (internal/modules/cjs/loader.js:825:5)
    at Object._register [as register] (/Users/daniel/code/quickapp-demo/node_modules/shelljs/src/common.js:453:12)
    at Object.<anonymous> (/Users/daniel/code/quickapp-demo/node_modules/shelljs/src/cat.js:4:8)
    at Module._compile (internal/modules/cjs/loader.js:1185:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1205:10)
    at Module.load (internal/modules/cjs/loader.js:1034:32)
    at Function.Module._load (internal/modules/cjs/loader.js:923:14)
    at Module.require (internal/modules/cjs/loader.js:1074:19)
    at require (internal/modules/cjs/helpers.js:72:18)
```

**Comment by Unitech** [comment]
When do you think it will be published?

**Comment by nfischer** [comment]
Thanks for raising this to my attention. I'll take a look and see if I can spin up a patch release.

**Comment by Stono** [comment]
Just adding my +1 to this, a lot of our child dependencies seem to use this module so its blocking many upgrades!  Thanks for your efforts fixing it

**Comment by Unitech** [comment]
Thanks @nfischer, can't wait to bump PM2 with newer shelljs version, for info trying the the shelljs patch on PM2 work as expected https://travis-ci.org/github/Unitech/pm2

**Comment by nfischer** [comment]
This should be released as v0.8.4 :shipit: :tada: This only differs from v0.8.3 by pulling in the referenced change.

2 tests fail on the `0.8-release` branch, but I'm not concerned:

* `exec › set maxBuffer (very small)` - This was a node v10 change (see #915), and I don't think we need to backport the workaround
* `exec › command that fails` - This test uses `shx@0.2.2`, which pulls in its own copy of shelljs@v0.7.8. So this is just a test issue.
